News/Controllers/UsersController.cs

```diff
@@ -24,14 +26,6 @@ public async Task<IActionResult> GetUserById([FromRoute] Guid id)
         return Ok(user);
     }
 
```

Comment:
If this is CRUD, then methods are missing: create & delete

Reply:
Create is Register method now. Delete method will be added if needed.